scripts/xjoin-config/debezium.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "xjoin.inventory.db",
```

Comment:
All of these parameters are mandatory?

Reply:
Yes, they are necessary. Here is the link to the documentation - https://debezium.io/documentation/reference/stable/connectors/postgresql.html#postgresql-connector-properties
BTW, I have copied them from ephemeral environment.